### PR TITLE
github: Extensive tests on master/PR review

### PR DESCRIPTION
**Pull request by JohannesLorenz**
Let's see what CI has to say...

### CODE REVIEW

**Review by friedolino78 — APPROVED** [review]
Ok

**Review by friedolino78 — APPROVED** [review]
2nd

**Comment by friedolino78** [comment]
3rd

**Review by friedolino78 — APPROVED** [review]
3rd

File: .github/workflows/ccpp.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: Ubuntu Build CI
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  pull_request_review:
+    types: [submitted]
```
Comment (fundamental) [inline .github/workflows/ccpp.yml:7]: Test comment as a review

**Review by fundamental — APPROVED** [review]
A different type of test comment?

**Pull request merged**